datalab/mlalpha/_job.py

```diff
@@ -68,6 +51,61 @@ def describe(self):
     job_yaml = yaml.safe_dump(self._info, default_flow_style=False)
     print job_yaml
 
+  @staticmethod 
+  def submit_training(job_request, job_id=None):
+    """Submit a training job.
+
+    Args:
+      job_request: the arguments of the training job in a dict. For example,
+          {
+            'package_uris':  'gs://my-bucket/iris/trainer-0.1.tar.gz',
+            'python_module': 'trainer.task',
+            'scale_tier': 'BASIC',
+            'region': 'us-central1',
+            'args': {
+              'train_data_paths': ['gs://mubucket/data/features_train'],
+              'eval_data_paths': ['gs://mubucket/data/features_eval'],
+              'metadata_path': 'gs://mubucket/data/metadata.yaml',
+              'output_path': 'gs://mubucket/data/mymodel/',
+            }
+          }
+      job_id: id for the training job. If None, an id based on timestamp will be generated.
+    Returns:
+      A Job object representing the cloud training job.
+    """
+    new_job_request = dict(job_request)
+    # convert job_args from dict to list as service required.
+    if 'args' in job_request and isinstance(job_request['args'], dict):
+      job_args = job_request['args']
+      args = []
+      for k,v in job_args.iteritems():
+        if isinstance(v, list):
+          for item in v:
+            args.append('--' + k)
```

Comment:
k is the name of the arg so it should always be a string. I added str() to be safe.